Formula/p/pgcopydb.rb

```diff
@@ -0,0 +1,36 @@
+class Pgcopydb < Formula
+  desc "Copy a Postgres database to a target Postgres server"
+  homepage "https://github.com/dimitri/pgcopydb"
+  url "https://github.com/dimitri/pgcopydb/archive/refs/tags/v0.17.tar.gz"
+  sha256 "7ed96f7bbc0a5250f3b73252c3a88d665df9c8101a89109cde774c3505882fdf"
+  license "PostgreSQL"
+  head "https://github.com/dimitri/pgcopydb.git", branch: "main"
+
+  depends_on "sphinx-doc" => :build
+  depends_on "bdw-gc"
+  depends_on "libpq"
+
+  def install
+    system "make", "bin"
+    bin.install "src/bin/pgcopydb/pgcopydb"
+    system "make", "-C", "docs", "man"
+    man1.install Dir["docs/_build/man/*"]
+  end
+
+  def caveats
+    <<~EOS
+      PostgreSQL client CLI tools must be installed for `pgcopydb` to work.
+
+      If you want to use the included PostgerSQL client CLI tools
+      add the following to your shell profile e.g. ~/.profile or ~/.zshrc:
+        export PATH="#{HOMEBREW_PREFIX}/opt/libpq/bin:$PATH"
+    EOS
+  end
+
+  test do
+    ENV["PATH"] = "#{HOMEBREW_PREFIX}/opt/libpq/bin:#{ENV["PATH"]}"
```

Comment:
This is configured by default 
```suggestion
```

Reply:
This is not configured by default because `libpq` is `keg_only`. So in order to use Postgres CLI tools from `libpq` we need to add this manually.

Reply:
Then it should be 
```suggestion
    ENV.prepend_path "PATH", Formula["libpq"].opt_bin
```

Reply:
The wrapper script is a great idea! Created it